pydoro/pydoro_tui.py

```diff
@@ -90,6 +92,25 @@ def draw():
 
 
 def main():
+    # Parse arguments
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--focus", help="focus mode: hides clock and \
+                        mutes sounds (equivalent to --no-clock and --no-sound)",
+                        action="store_true")
+    parser.add_argument("--no-clock", help="hides clock",
+                        action="store_true")
+    parser.add_argument("--no-sound", help="mutes all sounds",
+                        action="store_true")
+    args = parser.parse_args()
+
+    # Check for no-clock (or focus mode)
+    if args.no_clock or args.focus:
+        tomato._no_clock = True
+
+    # Check for no-sound (or focus mode)
+    if args.no_sound or args.focus:
+        tomato._no_sound = True
```

Comment:
It's kinda weird to set a private attribute, no? Isn't that why we created it? Can you initialize the tomato object with it's config?

Reply:
I did change it to non private actually :)